source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/Application.java

```diff
@@ -147,6 +147,15 @@ public int run(final String[] args) {
          */
         ProductInformation.initialize(ProductName.CLC);
 
+        // We need to check for the -notelemetry option before we send any
+        // telemetry to the server
+        for (final String arg : args) {
+            if (arg.equalsIgnoreCase("-notelemetry") || //$NON-NLS-1$
+                arg.equalsIgnoreCase("/notelemetry")) { //$NON-NLS-1$
+                TfsTelemetryHelper.disableSendingTelemetry();
+            }
+        }
+
```

Comment:
A command line option (even an undocumented one) seemed like a better approach. Do we use environment variables in other places like this?